core/arch/arm/include/mm/core_mmu.h

```diff
@@ -481,6 +481,8 @@ static inline TEE_Result cache_op_outer(enum cache_op op __unused,
 /* Check cpu mmu enabled or not */
 bool cpu_mmu_enabled(void);
 
+void map_memarea_sections(const struct tee_mmap_region *mm, uint32_t *ttb);
```

Comment:
This is core_mmu_v7.c, specific not LPAE. A generic `cpu_mmu_map_memarea()` could direct to the mmu_v7/lpae map functions. Lpae would be not supported at first I guess.
minor: can add a description since the function is now exported.

Reply:
I do not have platforms with LPAE enabled. I'll add a description.